.circleci/config.yml

```diff
@@ -30,68 +27,62 @@ aliases:
             poetry run ./.circleci/add-known-hosts.sh
             chmod +x .circleci/github-pages.sh
             poetry run ./.circleci/github-pages.sh
+          environment:
+            REPOSITORY_PATH: /home/circleci/docs
+            DEPLOYMENT_GIT_SSH: git@github.com:Sceptre/sceptre.github.io.git
```

Comment:
i don't see these ENV Vars used anywhere?

Reply:
These lines were just moved from above (they're used in the `github-pages.sh` script) https://github.com/Sceptre/sceptre/blob/4361e7d8b0b942406ec663e622647bc875a4faff/.circleci/config.yml#L14-L16

Reply:
docs job failed :(   https://app.circleci.com/pipelines/github/Sceptre/sceptre/2034/workflows/885ac577-7398-406f-a33d-3542580761cd/jobs/11094
```
Building docs in /home/circleci/docs/*******.github.io/dev
./.circleci/github-pages.sh: line 63: sphinx-apidoc: command not found
```

Reply:
@zaro0508 the step to install with `--all-extras` has changed, and I think that option is being ignored now. `sphinx-apidoc` is part of the optional sphinx package. I'm still investigating.

Reply:
I tracked down the step "Install dependencies with poetry using project pyproject.toml" in the [build job 11085](https://app.circleci.com/pipelines/github/Sceptre/sceptre/2034/workflows/885ac577-7398-406f-a33d-3542580761cd/jobs/11085).

The step command is `poetry install --no-ansi --all-extras`. The output shows that it installs the `sphinx` package.

```
  • Installing sphinx (5.1.1)
```

So that's not the problem.

Reply:
oops, fixed (I had remove the `restore_cache` step since I didn't think it needed it)

Reply:
Glad it was something "simple" like the cache. I had resorted to reading the Oython orb source to figure out what was going on.

The docs job looks like a good one to simplify after we complete this work.

thanks!